AntiTranslate.user.js

```diff
@@ -1,5 +1,5 @@
 // ==UserScript==
-// @name         Youtube title translate reverser
+// @name         Youtube title fixer
```

Comment:
The title of someone else's work probably shouldn't be changed, although if I had to change it, it would probably be to follow the repo's name, `Youtube Autotranslate Canceler`.